tests/helpers/acceptance.js

```diff
@@ -83,7 +84,9 @@ function linkDependencies(projectName) {
   let nodeManifest = fs.readFileSync(path.join(runFixture, 'package.json'));
 
   let packageCache = new PackageCache(root);
-  packageCache.create('node', 'yarn', nodeManifest, [{ name: 'ember-cli', path: root }]);
+  let packager = which.sync('yarn', { nothrow: true }) ? 'yarn' : 'npm';
```

Comment:
I guess it would make sense to only run this once for the whole test suite. Can you extract the `which.sync('yarn', { nothrow: true })` code to e.g. `tests/helpers/has-yarn.js` and export the result from that file?